CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -692,8 +693,8 @@ local function setBuyMoreRobuxDialog(playerBalance)
 	local neededRobux = PurchaseData.CurrencyAmount - playerBalanceInt
 	local productInfo = PurchaseData.ProductInfo
 
-	local descriptionText = "You need "..formatNumber(neededRobux).." more ROBUX to buy the "..productInfo["Name"].." "..
-		ASSET_TO_STRING[productInfo["AssetTypeId"]]
+	local descriptionText = "You need %s more ROBUX to buy the %s %s"
+	descriptionText = string.format(descriptionText, formatNumber(neededRobux), formatNumber(productInfo["Name"]), ASSET_TO_STRING[productInfo["AssetTypeId"]] or "")
```

Comment:
does formatNumber work for productInfo["Name"]? or at the least it seems a little weird